src/main/java/uk/gov/ons/ssdc/caseprocessor/model/entity/SmsTemplate.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ons.ssdc.caseprocessor.model.entity;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+
+@Data
+@Entity
+public class SmsTemplate {
+  @Id private String packCode;
+
+  @Type(type = "jsonb")
+  @Column(columnDefinition = "jsonb", nullable = false)
+  private String[] template;
+
+  @Column(nullable = false)
+  private UUID notifyId;
```

Comment:
what is the `notifyId` for?

Reply:
is it the notify template ID? If so, it should probably have the word "template" in the name